back/spec/services/content_image_service_spec.rb

```diff
@@ -0,0 +1,213 @@
+require 'rails_helper'
+
+describe ContentImageService do
+  before do
+    @text_image_1, @text_image_2 = create_list :text_image, 2
+    allow(TextImage).to receive(:create!).and_return(@text_image_1, @text_image_2)
+  end
+
+  let(:subclass) do
```

Comment:
I like to discuss this way of testing, because it is not clear what is being tested. Are the tests testing the subclass, which does not exist in production, or do they test something else.?